packages/components/src/components/teaser/teaser.stories.ts

```diff
@@ -382,3 +382,361 @@ export const Parts = {
     });
   }
 };
+
+/**
+ * ## Not clickable teaser
+ *
+ * This is the default state of the teaser. The teaser itself is not clickable, but links can be placed inside.
+ * There is no need for any extra steps, the teaser can be used as it is.
+ *
+ * ```html
+ * <sd-teaser>
+ *   <sd-button href="#">Link</sd-button>
+ * </<sd-teaser>
+ * ```
+ */
+
+export const SamplesNotClickable = {
+  name: 'Samples: Not clickable teaser',
+  parameters: {
+    controls: {
+      disable: true
+    },
+    backgrounds: {
+      default: 'white'
+    }
+  },
+  render: () => {
+    return html`
+      <style>
+        #teaserWithContentPlaceholder::part(media) {
+          flex-grow: 1;
+        }
+      </style>
+      <div class="flex justify-between gap-8">
+        <sd-teaser variant="primary" breakpoint="9999" inset class="flex-1">
+          <div slot="media" class="relative">
+            <img class="aspect-video object-cover" src="./placeholders/generic.jpg" alt="A generic placeholder jpg" />
```

Comment:
To ensure full accessibility for the image, you would need to describe what is depicted in the image. For example, A **generic placeholder image where a father and son are playing football.**

Reply:
In this case it's really just a 'A generic placeholder jpg', or should I use another placeholder image?
![image](https://github.com/solid-design-system/solid/assets/76621625/35a09470-2e4e-4591-94b1-56cf738514c8)

Reply:
Ah, okay! No than its not necessary.